requirements.txt

```diff
@@ -12,3 +12,4 @@ sphinxcontrib-napoleon
 setuptools_scm
 setuptools_scm_git_archive
 nbsphinx
+bam2fasta
```

Comment:
maybe I should remove this, should we still keep bam2fasta optional? @olgabot @luizirber